mne/annotations.py

```diff
@@ -900,6 +903,9 @@ def events_from_annotations(raw, event_id="auto",
         The events.
     event_id : dict
         The event_id variable that can be passed to Epochs.
+    event_duration : ndarray, shape (n_events,)
+        Event durations (in samples). This is returned only if
+        ``return_durations=True``.
```

Comment:
I don't like that here a duration is now in samples. it's confusing.

Reply:
Event onsets are also in samples (and have always been).

Reply:
I don't like it either. 

WDYT of `extend` or `span` ?